core/services/vrf/go_generate_test.go

```diff
@@ -216,5 +220,30 @@ func compileCommand(t *testing.T) string {
 		}
 		panic(err)
 	}
-	return string(cmd)
+	return strings.Trim(string(cmd), "\n")
+}
+
+func boxOutput(s string, args ...interface{}) string {
+	s = fmt.Sprintf(s, args...)
```

Comment:
What is `s`?

Reply:
Seems to be the error string.

Reply:
Trying to discourage the use of single letter variables here in code, except for one liners - maybe could use a better name, e.g: `errorMsg` or `description`

Reply:
It is one of the idiomatic Go practices that I dislike...

Reply:
I've always thought the idiomatic go practice was to use one letter vars only for `this` like or for one liners, everywhere else I see longer variables.

Reply:
Not so exact, but it's along those lines:

> Variable names in Go should be short rather than long. This is especially true for local variables with limited scope. Prefer c to lineCount. Prefer i to sliceIndex.
>
> The basic rule: the further from its declaration that a name is used, the more descriptive the name must be. For a method receiver, one or two letters is sufficient. Common variables such as loop indices and readers can be a single letter (i, r). More unusual things and global variables need more descriptive names.

https://github.com/golang/go/wiki/CodeReviewComments#variable-names

Reply:
Close enough to what I said :D